lib/gcloud/bigquery/table.rb

```diff
@@ -700,29 +708,20 @@ def extract extract_url, options = {}
       #
       # === A note about large direct uploads
       #
-      # You may encounter a Broken pipe (Errno::EPIPE) error when attempting to
```

Comment:
All this documentation should stay. It looks like your master was out of date when you created the branch for this pull request. Can you pull the latest from master and then rebase your commit on that?